ui/pages/Project/components/PhenotypePrioritizedGenes.jsx

```diff
@@ -27,7 +28,12 @@ const PHENOTYPE_GENE_INFO_COLUMNS = [
       />
     ),
   },
-  { name: 'tool', width: 1, content: 'Tool' },
+  {
+    name: 'tool',
+    width: 1,
+    content: 'Tool',
+    format: ({ tool }) => PHENOTYPE_PRIORITIZATION_FIELDS[tool],
```

Comment:
what is this changing?

Reply:
this is related to my comment here: https://github.com/broadinstitute/seqr/pull/4056/files#r1588101844

Reply:
removed this change from this PR 😆